app/service/url_shortener.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import APIRouter, HTTPException
+from pydantic import BaseModel
+from app.models.database import create_link, get_link_by_short_url
+
+#this file handles the logic of interacting and modifying the database
+router = APIRouter()
+
+#ShortenRequest ensures the input is in the correct format (JSON)
+class ShortenRequest(BaseModel):
+    original_url: str
+
+
+@router.post("/shorten-url")
```

Comment:
it looks like this file is a duplicate of `api_handlers.py`. I think you can remove this one.